polkadot/runtime/rococo/src/lib.rs

```diff
@@ -1799,6 +1802,9 @@ sp_api::impl_runtime_apis! {
 			parachains_staging_runtime_api_impl::disabled_validators::<Runtime>()
 		}
 
+		fn client_features() -> ClientFeatures {
```

Comment:
I would be in favor of extracting this API in a new PR and merging it without waiting for the rest of this changes to come together. I would like this use this here:https://github.com/paritytech/polkadot-sdk/issues/628

Reply:
I agree. I can work on this so that we can both use it in our PRs